src/peft/tuners/lora/config.py

```diff
@@ -57,6 +57,10 @@ class LoraConfig(PeftConfig):
         bias (`str`): Bias type for Lora. Can be 'none', 'all' or 'lora_only'. If 'all' or 'lora_only', the
             corresponding biases will be updated during training. Be aware that this means that, even when disabling
             the adapters, the model will not produce the same output as the base model would have without adaptation.
+        use_rslora (`bool`):
+            When set to True, uses <a href='https://doi.org/10.48550/arXiv.2312.03732'>Rank-Stabilized LoRA</a> which
```

Comment:
```suggestion
            When set to True, uses <a href='https://doi.org/10.48550/arXiv.2312.03732'> Rank-Stabilized LoRA</a> which
```

For some reason, there is no space before the name, see here:

https://moon-ci-docs.huggingface.co/docs/peft/pr_1244/en/package_reference/lora

Maybe this fixes it.

Reply:
I think I may have already fixed it [in the last commit](https://github.com/huggingface/peft/pull/1244/commits/44e6eabed5598090e3c5be84802b908e73f477d1) by making sure there is a space and not a newline after "uses". My main problem is I can't test the compilation of the docs as I'm still trying to get doc-builder working for me (it compiles the docs with no changes for me for some reason)

Reply:
The docs should be updated automatically (once the "Build PR Documentation" job has finished). But anyway, the space is now there.